Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
+    method_source (0.9.2)
+    mustermann (1.0.3)
     pg (0.20.0)
-    pry (0.10.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
-    pry-byebug (3.4.2)
-      byebug (~> 9.0)
+      method_source (~> 0.9.0)
+    pry-byebug (3.6.0)
+      byebug (~> 10.0)
       pry (~> 0.10)
-    rack (2.0.3)
-    rack-protection (2.0.0)
+    rack (2.0.6)
+    rack-protection (2.0.4)
       rack
-    rack-test (0.6.3)
-      rack (>= 1.0)
-    rake (12.0.0)
-    rspec (3.6.0)
-      rspec-core (~> 3.6.0)
-      rspec-expectations (~> 3.6.0)
-      rspec-mocks (~> 3.6.0)
-    rspec-core (3.6.0)
-      rspec-support (~> 3.6.0)
-    rspec-expectations (3.6.0)
+    rack-test (1.1.0)
```

Comment:
### [rack-test](http://github.com/rack-test/rack-test)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 0.6.3 → 1.1.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **82** commits. See the full changes on [the compare page](https://github.com/rack-test/rack-test/compare/v0.6.3...v1.1.0).

These are the first 10 commits:
- (4a4b2c1) [Don't clobber uri.query when it's present.](https://github.com/rack-test/rack-test/commit/4a4b2c1bde9ce00929df2d5458915e342b9c1ea1)
- (55c1d95) [Fix Rakefile to only run rack-test specs.](https://github.com/rack-test/rack-test/commit/55c1d95c12e5f5c9524998c897c5836584f820fa)
- (56ddeda) [Merge pull request #77 from adamtanner/master](https://github.com/rack-test/rack-test/commit/56ddeda4ea8580d188c1eda43933b9ba8ed33dbc)
- (5f78451) [Fix creation of tempfiles in Rack::Test::UploadedFile](https://github.com/rack-test/rack-test/commit/5f7845118f649b60862a9bc01f6b2a0f932e89d7)
- (0cd8537) [Merge pull request #129 from szalansky/fix_creation_of_tempf…](https://github.com/rack-test/rack-test/commit/0cd85371dd5a347be78e48559b280ced72fab7f9)
- (7536f85) [Update to latest rspec version and syntax.](https://github.com/rack-test/rack-test/commit/7536f85a7b50181cc005c31fead986a69c61d981)
- (d016695) [Do not set Content-Type for DELETE requests.](https://github.com/rack-test/rack-test/commit/d016695d84dd2e9b09eb77e5a89ffcbc1892921a)
- (a519e56) [Improve Cookie testing.](https://github.com/rack-test/rack-test/commit/a519e5615b571d7a34d92c68da5ccfffeef0822c)
- (fed7b5d) [Add License information to gemspec.](https://github.com/rack-test/rack-test/commit/fed7b5d6b6143847401870ec7d441806d00fb903)
- (c6dc62c) [Make `build_multipart` work without mixing in Rack::Test::Ut…](https://github.com/rack-test/rack-test/commit/c6dc62cfee7d10348eed46beb7f0ef252c5f5248)


</details>